rome/src/test/java/com/rometools/rome/io/impl/NumberParserTest.java

```diff
@@ -0,0 +1,66 @@
+package com.rometools.rome.io.impl;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+
+public class NumberParserTest {
+
+    @Test
+    public void testLongParseFailure() {
+        Long num = NumberParser.parseLong("Non Long");
+        assertNull(num);
+    }
+
+    @Test
+    public void testLongParseSuccess() {
+        Long num = NumberParser.parseLong("1");
+        assertTrue(num == 1L);
+    }
+
+    @Test
+    public void testLongParseDefaultSuccess() {
+        Long num = NumberParser.parseLong("Non Long", 1L);
```

Comment:
This method returns primitive `long`, no need for boxing. Same for other methods like this.